main.py

```diff
@@ -31,15 +20,14 @@ def print_card_special_options():
 print("====== START ======\n")
 
 player = Player()
-kos_hp = 5
+npc = Player()
```

Comment:
I'm guessing I am probably just missing something, but how is this equivalent? kos_hp becomes npc = the Player function?

Reply:
Here we are creating an instance of the `Player` class for the NPC in our `main.py` file instead of simply keeping track of the hit points using a local var in this file. We can use this instance to keep track of both the npc's health and whether it is alive. This is the same thing that we did previously with the player. It's a little confusing at the moment because they are both instances of a class called "Player" instead of something like "Combatant" or "Participant" or "Belligerent" (to imagine a few alternatives). For now, I decided to use that class for both the player and the NPC for now because both currently have the same properties and method(s) as far as the program is concerned -- they both have hit points, an "Am I alive?" status, and can both receive damage. We ought to consider a name change for the Player class so that it aptly applies to both the "good guys" and the "bad guys". However, if at some point in the future the properties and methods needed for players and NPCs differ greatly enough, then it would likely make sense to split these out into separate classes so that their functionality is better encapsulated and easier to unit test.

Reply:
To be clear, I do think that we should change the name of `Player` class to something else but I'll leave it up to you to give a suggestion.  Remember that code is sand and we can very easily change it later if/when we split out separate classes for the player and NPCs.

Reply:
Collectively decided to change class and module names to Character.